ui/components/app/modals/metametrics-opt-in-modal/metametrics-opt-in-modal.component.js

```diff
@@ -27,76 +27,81 @@ export default class MetaMetricsOptInModal extends Component {
-                how our users interact with the extension. This data will be
-                used to continually improve the usability and user experience of
-                our product and the Ethereum ecosystem.
+                {t('metametricsOptInDescription')}
               </div>
               <div className="metametrics-opt-in__description">
-                MetaMask will..
+                {t('metametricsCommitmentsIntro')}
               </div>
 
               <div className="metametrics-opt-in__committments">
                 <div className="metametrics-opt-in__row">
                   <i className="fa fa-check" />
                   <div className="metametrics-opt-in__row-description">
-                    Always allow you to opt-out via Settings
+                    {t('metametricsCommitmentsAllowOptOut')}
                   </div>
                 </div>
                 <div className="metametrics-opt-in__row">
                   <i className="fa fa-check" />
                   <div className="metametrics-opt-in__row-description">
-                    Send anonymized click & pageview events
-                  </div>
-                </div>
-                <div className="metametrics-opt-in__row">
-                  <i className="fa fa-check" />
-                  <div className="metametrics-opt-in__row-description">
-                    Maintain a public aggregate dashboard to educate the
-                    community
+                    {t('metametricsCommitmentsSendAnonymizedEvents')}
                   </div>
                 </div>
                 <div className="metametrics-opt-in__row metametrics-opt-in__break-row">
                   <i className="fa fa-times" />
                   <div className="metametrics-opt-in__row-description">
-                    <span className="metametrics-opt-in__bold">Never</span>{' '}
-                    collect keys, addresses, transactions, balances, hashes, or
-                    any personal information
+                    {t('metametricsCommitmentsNeverCollectKeysEtc', [
+                      <span
+                        className="metametrics-opt-in__bold"
+                        key="neverCollectKeys"
+                      >
+                        {t('metametricsCommitmentsBoldNever')}
+                      </span>,
+                    ])}
                   </div>
                 </div>
                 <div className="metametrics-opt-in__row">
                   <i className="fa fa-times" />
                   <div className="metametrics-opt-in__row-description">
-                    <span className="metametrics-opt-in__bold">Never</span>{' '}
-                    collect your full IP address
+                    {t('metametricsCommitmentsNeverCollectIP', [
+                      <span
+                        className="metametrics-opt-in__bold"
+                        key="neverCollectKeys"
```

Comment:
Nit: probably better to use a different `key` name here, just so it doesn't confuse readers

```suggestion
                        key="neverCollectIP"
```

Reply:
Updated both here and in the full page component